mantidproject/mslice
# Improved packaging and configuration

**Description of work:**

- Modernized the project structure by moving mslice code into a src folder and moving tests to the same level as src. 
- Moved most of the configuration into a pyproject.toml file (setup.cfg is only used for nosetests from now on). 
- Cleaned up unused files


**To test:**

Check that unit tests are passing. Check that mslicedevel scripts are still working. Once merged into main check the nightly conda package from the next day on and also the online documentation (it should show the version as 2.6 instead of 2.6.0.

Fixes #857
